utils/getRoutes.js

```diff
@@ -11,3 +10,15 @@ export default async () => {
 
   return articles
 }
+
+export const getFeatureRoutes = async (params) => {
+  const {
+    data: { data },
+  } = await axios.get(`https://cms.formester.com/api/features?populate=deep`)
+
+  const features = data.map((item) => {
+    return `/features/${item.slug}`
+  })
+
+  return features
+}
```

Comment:
Don't we need to handle errors?

Reply:
It will only be called only during the build time.